config/environments/production.rb

```diff
@@ -76,4 +76,10 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.delivery_method = :mailgun
+  config.action_mailer.mailgun_settings = {
+  api_key: ENV["MAILGUN_API_KEY"],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.